63 main (#2)
openid-foundation-japan/800-63-4: _sp800-63/sec2_introduction.ja.md

Digital Identity は, デジタルサービスのコンテキストでは常に一意であるが, すべてのコンテキストで常に個人を一意に特定できるわけではない. さらに, Digital Identity はデジタルサービスのコンテキスト内では一意で明確な意味を持つが, Digital Identity の背後にある個人の現実の身元は知られていない場合がある. 本書では, 「人」は自然人のみを指す（すなわち, 法人を指していない）. 

<!-- Establishing a digital identity is intended to demonstrate trust between the holder of the digital identity and the person, organization, or system on the other side of the digital transaction. However, this process can present challenges. As in relationships and transactions in the physical world, there are multiple opportunities for mistakes, miscommunication, impersonation, and other attacks that fraudulently claim another person's digital identity. Additionally, given the broad range of individual needs, constraints, capacities, and preferences, digital services must be designed with equity and flexibility in mind to ensure broad and enduring participation. -->

> Digital Identity の確立は, Digital Identity の保有者と, デジタル取引の相手側にいる個人, 組織, または システムとの間に信頼を示すことを意図している. しかし, このプロセスには課題がある. 物理的な世界での関係や取引と同様に, ミスコミュニケーション, なりすまし, 他人の Digital Identity の詐称, およびその他の攻撃の機会が多数存在する. 加えて, 個人のニーズ, 制約, 能力, 嗜好が多岐に渡ることを考えると, デジタルサービスは, 広く永続的に参加できるように, Equity と柔軟性を考慮して設計する必要がある. 

Reviewer: ```suggestion
Digital Identity の確立は, Digital Identity の保有者と, Digital Transaction の相手側にいる個人, 組織, または システムとの間に信頼を示すことを意図している. しかし, このプロセスには課題がある. 物理的な世界での関係や Transaction と同様に, ミスコミュニケーション, なりすまし, 他人の Digital Identity の詐称, およびその他の Attack の機会が多数存在する. 加えて, 個人のニーズ, 制約, 能力, 嗜好が多岐に渡ることを考えると, デジタルサービスは, 広く永続的に参加できるように, Equity と柔軟性を考慮して設計する必要がある. 
```

<!-- These guidelines lay out a model for federal programs and other organizations to assess and manage risks associated with digital identity systems, including the processes, policies, data, people, and technologies that support digital identity management. The model is supported by a series of processes: identity proofing, authentication, and federation. The identity proofing process establishes that a subject is a specific physical person. The digital authentication process determines the validity of one or more authenticators used to claim a digital identity and establishes confidence that a subject attempting to access a digital service: (1) is in control of the technologies being used for authentication, and (2) is the same subject that previously accessed the service. Finally, the federation process allows for identity information to be shared in support of authentication across systems. -->

> これらのガイドラインは, 連邦プログラムおよびその他の組織が, Digital Identity 管理をサポートするプロセス, ポリシー, データ, 人, および技術を含む Digital Identity システムに関連するリスクを評価および管理するためのモデルを示している. このモデルは, Identity Proofing, Authentication, および Federation という一連のプロセスによってサポートされている. Identity Proofing は, 対象が特定の物理的人物であることを証明する. Digital Authentication プロセスは, Digital Identity を主張するために使用される 1 つまたは複数の Authenticator の有効性を判断し, デジタルサービスにアクセスしようとする対象者が, （1） Authentication に使用されている技術を管理しており, （2）以前にサービスにアクセスした対象者と同一であるという信頼を確立するものである. 最後に, Federation プロセスは, システム間で Authentication をサポートするために Identity 情報を共有することを可能にする. 

Reviewer: ```suggestion
これらのガイドラインは, 連邦プログラムおよびその他の組織が, Digital Identity 管理をサポートするプロセス, ポリシー, データ, 人, および技術を含む Digital Identity システムに関連するリスクを評価および管理するためのモデルを示している. このモデルは, Identity Proofing, Authentication, および Federation という一連のプロセスによってサポートされている. Identity Proofing は, Subject が特定の物理的人物であることを証明する. Digital Authentication プロセスは, Digital Identity を主張するために使用される 1 つまたは複数の Authenticator の有効性を判断し, デジタルサービスに Access しようとする Subject が, （1） Authentication に使用されている技術を管理しており, （2）以前にサービスにアクセスした Subject と同一であるという信頼を確立するものである. 最後に, Federation プロセスは, システム間で Authentication をサポートするために Identity 情報を共有することを可能にする. 
```

> さらに, 本書は Credential Service Provider（CSP）, Verifier , および Relying Party (RP) に対する指針を提供し, Digital Identity サービスの実装のために組織が従うべき Risk Management プロセスを説明し, *NIST Risk Management Framework* [[NISTRMF]](sec8_references.md#ref-NIST-RMF) およびその構成特別出版を補完するものである. この出版物は, NIST RMF を拡張して, Equity と Usability への配慮を Digital Identity Risk Management プロセスに組み込む方法を概説し, 企業の運用と資産だけでなく, 個人, 他の組織, およびより広く社会への影響を考慮することの重要性を強調している. さらに, Digital Authentication は, 個人の情報への不正アクセスのリスクを軽減することで Privacy 保護をサポートするが, Identity Proofing, Authentication, Authorization, および Federation は, しばしば個人情報の処理を伴うため, これらの機能は Privacy リスクを引き起こす可能性もある. したがって, このガイドラインは, 関連する潜在的な Privacy リスクの軽減を支援するための Privacy 要件および考慮事項を含んでいる. 

Reviewer: ```suggestion
さらに, 本書は Credential Service Provider（CSP）, Verifier , および Relying Party (RP) に対する指針を提供し, Digital Identity サービスの実装のために組織が従うべき Risk Management プロセスを説明し, *NIST Risk Management Framework* [[NISTRMF]](sec8_references.md#ref-NIST-RMF) およびそれを構成する Special Publication 群を補完するものである. この出版物は, NIST RMF を拡張して, Equity と Usability への配慮を Digital Identity Risk Management プロセスに組み込む方法を概説し, 企業の運用と資産だけでなく, 個人, 他の組織, およびより広く社会への影響を考慮することの重要性を強調している. さらに, Digital Authentication は, 個人の情報への不正アクセスのリスクを軽減することで Privacy 保護をサポートするが, Identity Proofing, Authentication, Authorization, および Federation は, しばしば個人情報の処理を伴うため, これらの機能は Privacy リスクを引き起こす可能性もある. したがって, このガイドラインは, 関連する潜在的な Privacy リスクの軽減を支援するための Privacy 要件および考慮事項を含んでいる. 
```

<!-- Finally, while this publication provides organizations with technical requirements and recommendations for establishing, maintaining, and authenticating the digital identity of subjects in order to access digital systems over a network, additional support options outside the purview of information technology teams may need to be provided to address barriers and adverse impacts, foster equity, and successfully deliver on mission objectives. -->

> 最後に, 本書は, ネットワーク上のデジタル・システムにアクセスするために対象者の Digital Identity を確立, 維持, および認証するための技術要件と推奨事項を組織に提供しているが, 障壁や悪影響に対処し, Equity を育み, ミッション目標を正常に達成するには, 情報技術チームの権限外のサポート プションを追加する必要がある場合がある. 

Reviewer: ```suggestion
最後に, 本書は, ネットワーク上のデジタル・システムにアクセスするために対象者の Digital Identity を確立, 維持, および Authenticate するための技術要件と推奨事項を組織に提供しているが, 障壁や悪影響に対処し, Equity を育み, ミッション目標を正常に達成するには, 情報技術チームの権限外のサポートオプションを追加する必要がある場合がある. 
```

すべてのデジタルサービスに Identity Proofing　または Authentication が必要なわけではないが, このガイダンスは利用者層（市民, ビジネス・パートナー, 政府機関など）に関係なく, 何らかのレベルの Digital Identity が必要なすべてのオンライン Transaction に適用される. 

<!-- These guidelines primarily focus on organizational services that interact with external users, such as citizens accessing public benefits or private sector partners accessing collaboration spaces. However, it also applies to federal systems accessed by employees and contractors. The *Personal Identity Verification (PIV) of Federal Employees and Contractors* standard [[FIPS201]](sec8_references.md#ref-FIPS201) and its corresponding set of special publications and organization-specific instructions, extend these guidelines for the federal enterprise, providing additional technical controls and processes for issuing and managing Personal Identity Verification (PIV) cards, binding additional authenticators as derived PIV credentials, and using federation architectures and protocols with PIV systems. -->

> 本ガイドラインは, 公共サービスにアクセスする市民やコラボレーションスペースにアクセスする民間セクターのパートナーなど, 外部ユーザーとやり取りする組織サービスを主に対象としている. しかし, 職員や契約社員がアクセスする連邦システムにも適用される. *Personal Identity Verification (PIV) of Federal Employees and Contractors* standard [[FIPS201]]（sec8_references.md#ref-FIPS201）および対応する一連の専門出版物と組織固有の説明書は, 連邦企業向けにこれらのガイドラインを拡張し, Personal Identity Verification（PIV）カードの発行と管理, 派生 PIV Credential としての追加 Authenticator のバインド, PIV システムでの Federation アーキテクチャおよびプロトコルの使用に関する追加の技術統制および手順を規定している. 

Reviewer: ```suggestion
本ガイドラインは, 公共サービスにアクセスする市民やコラボレーションスペースにアクセスする民間セクターのパートナーなど, 外部ユーザーとやり取りする組織サービスを主に対象としている. しかし, 職員や契約社員がアクセスする連邦システムにも適用される. *Personal Identity Verification (PIV) of Federal Employees and Contractors* standard [[FIPS201]]（sec8_references.md#ref-FIPS201）および対応する一連の Special Publication と組織固有の説明書は, 連邦企業向けにこれらのガイドラインを拡張し, Personal Identity Verification（PIV）カードの発行と管理, 派生 PIV Credential としての追加 Authenticator のバインド, PIV システムでの Federation アーキテクチャおよびプロトコルの使用に関する追加の技術統制および手順を規定している. 
```

本ガイダンスの対象とならない Transaction には, 44 U.S.C. § 3542(b)(2) に定義される国家安全保障システムに関連するものが含まれる. デジタルプロセスでさまざまなレベルの Digital Identity Assurance を必要とする民間部門組織, 州, 地方, および部族の政府は, 必要に応じてこれらの標準の使用を検討することができる. 

<!-- Additionally, these technical guidelines do not address the identity of subjects for physical access (e.g., to buildings), though some identities used for online transactions may also be used for physical access. Additionally, this revision of these guidelines does not explicitly address device identity, often referred to as machine-to-machine (such as router-to-router) authentication or interconnected devices, commonly referred to as the internet of things (IoT), although these guidelines are written to refer to generic subjects wherever possible to leave open the possibility for applicability to devices. Furthermore, these guidelines do not address authorization of access to Application Programming Interfaces (APIs) on behalf of subjects. -->

> 加えて, これらの技術ガイドラインは, オンライン Transaction に使用される一部の Identity が物理的アクセスに使用される場合があるが, （建物など）物理的アクセスに対する Subject の Identity は扱っていない. また, 今回の改訂では, 機器間（ルータ間など） Authentication や Internet of Things (IoT) と呼ばれる相互接続されたデバイスの Identity については明示的に触れていないが, デバイスへの適用可能性を残すため, 可能な限り一般的な Subject に言及するよう記述されている. さらに, Application Programming Interfaces (API) に対する Subject の代理については, 本ガイドラインでは触れていない. 

Reviewer: ```suggestion
加えて, オンライン Transaction に使用される一部の Identity は（建物など）物理的 Access の際にも使用される場合があるが, これらの技術ガイドラインは物理的 Access に対する Subject の Identity は扱っていない. また, 今回の改訂では, 機器間（ルータ間など） Authentication や Internet of Things (IoT) と呼ばれる相互接続されたデバイスの Identity については明示的に触れていないが, デバイスへの適用可能性を残すため, 可能な限り一般的な Subject に言及するよう記述されている. さらに, Application Programming Interfaces (API) に対する Subject の代理については, 本ガイドラインでは触れていない. 
```

<!-- The components of identity assurance detailed in these guidelines are as follows: -->

本ガイドライン群において詳しく扱う Identity Assurance の構成要素は以下の通りである.

<!-- * **IAL** refers to the identity proofing process. -->
<!-- * **AAL** refers to the authentication process. -->
<!-- * **FAL** refers to the federation process, when the RP is connected through a federated protocol. -->

* **IAL** は, Identity Proofing プロセスについて述べる.
* **AAL** は, Authentication プロセスについて述べる.
* **FAL** は, RP が Federated Protocol で接続されている場合の Federation プロセスについて述べる. 

<!-- > Note: When described generically or bundled, these guidelines will refer to IAL, AAL, and FAL as ***xAL***. -->

> 注: 本ガイドラインでは, IAL , AAL , FAL を総称して, ***xAL*** と表記する場合がある.

<!-- SP 800-63 is organized as the following suite of volumes: -->

SP 800-63 は以下のような一連の Vol. から構成される:

<!-- SP 800-63 *Digital Identity Guidelines*: Provides the risk assessment methodology and an overview of general identity frameworks, using authenticators, credentials, and assertions together in a digital system, and a risk-based process of selecting assurance levels. _SP 800-63 contains both normative and informative material._ -->

> SP 800-63 *Digital Identity Guidelines* :　SP 800-63 では, Risk Assesment の方法論, デジタルシステムにおける Authenticator, Credential, Assertion を利用した一般的な Identity Framework の概観, およびリスクベースプロセスに基づく各 Assurance Level の選択方法について述べる. _SP 800-63 には規範的な資料と情報的な資料の両方が含まれている.__

Reviewer: ```suggestion
SP 800-63 *Digital Identity Guidelines* :　SP 800-63 では, Risk Assesment の方法論, デジタルシステムにおける Authenticator, Credential, Assertion を利用した一般的な Identity Framework の概観, およびリスクベースプロセスに基づく各 Assurance Level の選択方法について述べる. _SP 800-63 には規範的な資料と情報的な資料の両方が含まれている._
```

<!-- [[SP800-63A]](../_sp800-63a/sec1_purpose.md#purpose){:latex-href="#ref-SP800-63A"}: Provides requirements for enrollment and identity proofing of applicants, either remotely or in person, that wish to gain access to resources at each of the three identity assurance levels (IALs). It details the responsibilities of Credential Service Providers (CSPs) with respect to establishing and maintaining subscriber accounts and binding authenticators (either CSP-issued or subscriber-provided) to the subscriber account.
_SP 800-63A contains both normative and informative material._ -->

> [[SP800-63A]](../_sp800-63a/sec1_purpose.md#purpose){:latex-href="#ref-SP800-63A"}: 3 つの Identity Assurance Level（IAL）ごとに, リソースへのアクセスを希望する申請者の登録と Identity Proofing の要件 を規定する. この文書では, 利用者アカウントの確立および維持, ならびに Subscriber アカウントへの Authenticator（CSP 発行または Subscriber 提供のいずれか）のバインドに関する Credential Service Providers (CSP) の責務について詳述している. 

Reviewer: ```suggestion
[[SP800-63A]](../_sp800-63a/sec1_purpose.md#purpose){:latex-href="#ref-SP800-63A"}: 3 つの Identity Assurance Level（IAL）ごとに, リソースへの Access を希望する Applicant の登録と Identity Proofing の要件を規定する. この文書では, Subscriber Account の確立および維持, ならびに Subscriber Account への Authenticator（CSP 発行または Subscriber 提供のいずれか）のバインドに関する Credential Service Providers (CSP) の責務について詳述している. 
```

_SP 800-63B contains both normative and informative material._-->

[[SP800-63B]](../_sp800-63b/sec1_purpose.md#purpose){:latex-href="#ref-SP800-63B"}: 3つの Authentication Assurance Level（AAL）のそれぞれで使用できる Authentication プロセス（ Authenticator の選択など）の種類に関する推奨事項を規定する. また, 紛失または盗難時の無効化など, Authenticator のライフサイクルに関する推奨事項を記載している. 

_SP 800-63B contains both normative and informative material._

<!-- [[SP800-63C]](../_sp800-63c/sec1_purpose.md#purpose){:latex-href="#ref-SP800-63C"}: Provides requirements on the use of federated identity architectures and assertions to convey the results of authentication processes and relevant identity information to an agency application. Further, this volume offers privacy-enhancing techniques to share information about a valid, authenticated subject, and describes methods that allow for strong multi-factor authentication (MFA) while the subject remains pseudonymous to the digital service. 
_SP 800-63C contains both normative and informative material._-->

> [[SP800-63C]](../_sp800-63c/sec1_purpose.md#purpose){:latex-href="#ref-SP800-63C"}: Authentication プロセスの結果および関連する Identity 情報を機関アプリケーションに伝達するための, Federated Identity Architecture および Assertion の使用に関する要件を規定している. さらに, この文書では, 有効な認証対象者に関する情報を共有するためのプライバシー強化技術を提供し, 対象者がデジタルサービスに対して匿名のままで強力な　Multi-factor Authentication（MFA）を可能にする方法について説明している. 

Reviewer: ```suggestion
[[SP800-63C]](../_sp800-63c/sec1_purpose.md#purpose){:latex-href="#ref-SP800-63C"}: Authentication プロセスの結果および関連する Identity 情報を機関アプリケーションに伝達するための, Federated Identity Architecture および Assertion の使用に関する要件を規定している. さらに, この文書では, 有効な認証済 Subject に関する情報を共有するためのプライバシー強化技術を提供し, Subject がデジタルサービスに対して匿名のままで強力な　Multi-factor Authentication（MFA）を可能にする方法について説明している. 
```

_SP 800-63C contains both normative and informative material._

## Enterprise Risk Management Requirements and Considerations {#ERMreqs}

<!-- Effective enterprise risk management is multidisciplinary by default and involves the consideration of a diverse set of factors and equities. In a digital identity risk management context, these factors include, but are not limited to, information security, privacy, equity, and usability. It is important for risk management efforts to weigh these factors as they relate not only to enterprise assets and operations but also to individuals, other organizations, and society more broadly. -->

> 効果的な企業 Risk Management は, 基本的に多分野にまたがるものであり, 多様な要因および公平性の考慮が必要である. Digital Identity Risk Management のコンテキストでは, これらの要因には, 情報セキュリティ, Privacy, Equity, および Usability が含まれるが, これらに限定されるものではない. Risk Management の取り組みでは, 企業の資産および業務だけでなく, 個人, 他の組織, および社会により広く関係するこれらの要因を考慮することが重要である.

Reviewer: ```suggestion
効果的な企業 Risk Management は, 基本的に多分野にまたがるものであり, 多様な要因および Equity の考慮が必要である. Digital Identity Risk Management のコンテキストでは, これらの要因には, 情報セキュリティ, Privacy, Equity, および Usability が含まれるが, これらに限定されるものではない. Risk Management の取り組みでは, 企業の資産および業務だけでなく, 個人, 他の組織, および社会により広く関係するこれらの要因を考慮することが重要である.
```

<!-- The controls and requirements encompassed by the identity, authentication, and federation assurance levels under these guidelines augment, but do not replace or alter, the information and information system controls as determined under FISMA and the RMF. -->

これらのガイドラインに基づく Identity, Authentication, および Federation Assurance Level に包含される管理 および要件は, FISMA および RMF に基づいて決定される情報および情報システムの管理を補完するが, 置き換えたり変更することはない. 

### Privacy

<!-- When designing, engineering, and managing digital identity systems, it is imperative to consider the potential of that system to create privacy-related problems for individuals when processing PII &mdash; a problematic data action &mdash; and the potential impact of the problematic data action should it occur. Additionally, by focusing on the privacy engineering objectives of predictability, manageability, and disassociability, organizations can determine the types of capabilities a given system may need to be able to demonstrate how organizational privacy policies and system privacy requirements have been implemented. -->

> Digital Identity システムを設計, エンジニアリング, および管理する場合, そのシステムが PII を処理するときに個人のプライバシー関連の問題を引き起こす可能性, 問題のあるデータ操作, および問題のあるデータ操作が発生した場合の潜在的な影響を考慮することが不可欠である. さらに, Predictability, Manageability, およびDisassociabilityというプライバシー工学の目標に注目することで, 組織は, 組織のプライバシーポリシーとシステムのプライバシー要件がどのように実装されているかを実証できるように, 特定のシステムに必要な機能の種類を決定することができる.

Reviewer: ```suggestion
Digital Identity システムを設計, エンジニアリング, および管理する場合, そのシステムが PII を処理 (Processing) するときに個人のプライバシー関連の問題を引き起こす可能性, 問題のあるデータ操作, および問題のあるデータ操作が発生した場合の潜在的な影響を考慮することが不可欠である. さらに, Predictability, Manageability, およびDisassociabilityというプライバシー工学の目標に注目することで, 組織は, 組織のプライバシーポリシーとシステムのプライバシー要件がどのように実装されているかを実証できるように, 特定のシステムに必要な機能の種類を決定することができる.
```

*Privacy Act of 1974, 2010 Edition*, [[PrivacyAct]](sec8_references.md#ref-PrivacyAct)は, 連邦機関が記録システムに保持する個人に関する情報の収集, 維持, 使用, 開示に関する一連の公正な情報プラクティスを定めたものである.

<!-- When designing and implementing digital identity management processes and systems, privacy risk assessments are required for PII processing under these guidelines. Such privacy risk assessments can be used to support Privacy Impact Assessments under *OMB Guidance for Implementing the Privacy Provisions of the E-Government Act of 2002* [[M-03-22]](sec8_references.md#ref-M-03-22) as well as to select controls from NIST Special Publication 800-53, *Security and Privacy Controls for Information Systems and Organizations* [[SP800-53]](sec8_references.md#ref-SP800-53). Further, each volume of 800-63 (63A, 63B, and 63C) contains a specific section providing detailed privacy requirements and considerations for the implementation of the processes, controls, and requirements presented in that volume. -->

> Digital Identity 管理プロセスおよびシステムを設計・実装する場合, プライバシー Risk Assessment はこれらのガイドラインに基づく PII 処理に必要となる. そのようなプライバシー Risk Assessment は, *OMB Guidance for Implementing the Privacy Provisions of the E-Government Act of 2002* [[M-03-22]](sec8_references.md#ref-M-03-22) に基づくPrivacy Impact Assessments の支援に使用できるほか, NIST Special Publication 800-53, *Security and Privacy Controls for Information Systems and Organizations* [[SP800-53]](sec8_references.md#ref-SP800-53) からコントロールを選択するためにも使用することができる. さらに, 800-63 の各巻（63A, 63B, 63C）には, その巻で提示されたプロセス, 管理, 要件の実装に関する詳細なプライバシー要件と考慮事項を示す特定のセクションが含まれている. 

Reviewer: ```suggestion
Digital Identity 管理プロセスおよびシステムを設計・実装する場合, プライバシー Risk Assessment はこれらのガイドラインに基づく PII Processing に必要となる. そのようなプライバシー Risk Assessment は, *OMB Guidance for Implementing the Privacy Provisions of the E-Government Act of 2002* [[M-03-22]](sec8_references.md#ref-M-03-22) に基づくPrivacy Impact Assessments の支援に使用できるほか, NIST Special Publication 800-53, *Security and Privacy Controls for Information Systems and Organizations* [[SP800-53]](sec8_references.md#ref-SP800-53) からコントロールを選択するためにも使用することができる. さらに, 800-63 の各巻（63A, 63B, 63C）には, その巻で提示されたプロセス, 管理, 要件の実装に関する詳細なプライバシー要件と考慮事項を示す特定のセクションが含まれている. 
```

このガイダンスの読者は, Digital Identity システムおよびプロセスを彼らのニーズを最もよく支援する方法で設計または運用する機会を特定するために, サービスを提供する集団が直面する既存の不公平を考慮することが推奨される. 読者はまた, Digital Identity システムの設計または運用によって, 集団間の格差を含め, これらの集団経験および結果に潜在的または実際の影響を与えることを考慮するよう推奨される. 

<!-- For federal agencies implementing these guidelines, EO 13985 directs federal agencies to identify underserved communities for the programs and services that they provide and to determine and address any systemic barriers to underserved communities to provide equitable access to those programs and services. In alignment with the direction set by EO 13985, federal agencies should determine potential barriers communities and individuals may face to enrollment in and access to online benefits and services. They should also  identify whether programmatic changes may be necessary to advance equity. -->

> このガイドラインを実施する連邦政府機関に対し, EO 13985 は, 連邦政府機関が提供するプログラムおよびサービスについて, 十分なサービスを受けていないコミュニティを特定し, それらのプログラムおよびサービスへの公平なアクセスを提供するために, 十分なサービスを受けていないコミュニティに対する制度的障害を特定し対処するよう指示している. EO 13985 の方向性に沿って, 連邦機関は, コミュニティや個人がオンライン給付やサービスへの登録やアクセスに直面する可能性のある障壁を判断するべきである. また, 公平性を高めるためにプログラムの変更が必要であるかどうかを確認する必要がある. 

Reviewer: ```suggestion
このガイドラインを実施する連邦政府機関に対し, EO 13985 は, 連邦政府機関が提供するプログラムおよびサービスについて, 十分なサービスを受けていないコミュニティを特定し, それらのプログラムおよびサービスへの公平なアクセスを提供するために, 十分なサービスを受けていないコミュニティに対する制度的障害を特定し対処するよう指示している. EO 13985 の方向性に沿って, 連邦機関は, コミュニティや個人がオンライン給付やサービスへの登録やアクセスに直面する可能性のある障壁を判断するべきである. また, Equity を高めるためにプログラムの変更が必要であるかどうかを確認する必要がある. 
```

### Usability

<!-- Usability refers to the extent to which a system, product, or service can be used by specified users to achieve specified goals with effectiveness, efficiency, and satisfaction in a specified context of use. -->

Usability とは, あるシステム, 製品, サービスを, 特定のユーザーが, 特定の使用状況において, 有効性, 効率性, 満足性をもって, 特定の目標を達成するために使用できる程度を指す. 

<!-- Similar to equity, usability requires an understanding of the people interacting with a digital identity system or process, as well as their unique goals and context of use. To provide an effective, efficient, and satisfactory experience, readers of this guidance should take a holistic approach to considering the interactions that each user will engage in throughout the process of enrolling in and authenticating to a service. Throughout the design and development lifecycle of a digital identity system or process, it is important to conduct usability evaluation with representative users performing realistic scenarios and tasks in appropriate context of use. -->

> Equity と同様に, Usability には Digital Identity システムまたはプロセスに触れる人々, および彼ら固有の目標や使用状況の理解が必要である. 効果的, 効率的, かつ満足のいく体験を提供するために, このガイダンスの読者は, サービスへの登録 および認証のプロセスを通じて各ユーザが行うインタラクションを考慮する全体的なアプローチを取る必要がある. Digital Identity システムまたはプロセスの設計および開発ライフサイクルを通じて, 適切な使用状況で現実的なシナリオおよびタスクを実行する代表的なユーザによるユーザビリティ評価を実施することが重要である. 

Reviewer: ```suggestion
Equity と同様に, Usability には Digital Identity システムまたはプロセスに触れる人々, および彼ら固有の目標や使用状況の理解が必要である. 効果的, 効率的, かつ満足のいく体験を提供するために, このガイダンスの読者は, Enrollment および Authentication のプロセスを通じて各ユーザが行うインタラクションを考慮する全体的なアプローチを取る必要がある. Digital Identity システムまたはプロセスの設計および開発ライフサイクルを通じて, 適切な使用状況で現実的なシナリオおよびタスクを実行する代表的なユーザに対する Usability 評価を実施することが重要である. 
```
